app/controllers/katello/api/v2/hosts_bulk_actions_controller.rb

```diff
@@ -236,6 +237,16 @@ def traces
       respond_for_index(:collection => collection, :template => '../../../api/v2/host_tracer/index')
     end
 
+    api :PUT, "/hosts/bulk/resolve_traces", N_("Resolve traces for one or more hosts")
+    param_group :bulk_params
+    param :trace_ids, Array, :required => true, :desc => N_("Array of Trace IDs")
+    def resolve_traces
+      traces = Katello::HostTracer.resolvable.where(id: params[:trace_ids])
```

Comment:
This code currently allows resolving traces for hosts other than --host-id !

The code should be something like: `@host.traces.resolvable.where(id: params[:trace_ids])`

And if `traces` ends up being empty, I think the API should throw an error